src/pemsa/graphics/pemsa_graphics_api.cpp

```diff
@@ -611,14 +615,18 @@ static int print(lua_State* state) {
 
 	}
 
+	// Remove .0 if the string is an integer number
```

Comment:
ended up adding the fix for number strings ending in ".0" that were getting printed